README.md

```diff
@@ -0,0 +1,37 @@
+# Doto
+
+Welcome to Doto. The open-source software (OSS) project for a smart scheduling calendar app. Doto is an online calendar and to-do app. It has all of the basic functionality of any calendar app and can be used to make to-do lists. It also has smart scheduling capabilities, meaning that if the user wants to do a task, it can input it into our app and the app will allocate this task in a suitable time in the user’s calendar. Doto uses Google to sign up, meaning that to use this app, the user must have a google account. The development of Doto is done using the M.E.R.N (MongoDB, Express, React, Node) tech stack (more info about the tech stack can be found in the Wiki).
+
+## Why is this project Useful?
+This project is useful as it has the basic functionalities of any normal calendar app, but also has the added functionality of smart scheduling. This is particularly useful for people who like to plan their day but find it difficult to plan it well, as the app finds the most suitable time for whatever task the user wants to accomplish. It also encourages people who don’t usually like to plan their day to start planning their day as all they need to do is input the task and the app will choose the perfect time, no effort required. 
+
+
+## Development Setup
+It is recommended that you use **VsCode** when contributing to this project. Please install the **Eslint** and **Prettier** extensions so that the code style matches what has already been done before.
+
+## Running the code
+We are using npm for this particular project (https://www.npmjs.com/get-npm).
+### `cd doto-frontend/backend`
+(change to either frontend or backend depending on where you wish to contribute)
+
+### `npm install`
+
+### `npm start`
+
+Please check the front end [readme] (https://github.com/se701g2/Doto/blob/master/doto-frontend/README.md) for more information on running the front end of the code.
+
+## Usage Examples
+(will do once more of the website is done)
+
+## Contributing to this Project
+Please refer to our [wiki] (https://github.com/se701g2/Doto/wiki) for an outline of our coding conventions and git workflow
+
+## Meta
+Se701 Group2 - Se701group2@gmail.com
+Distributed under the MIT licence. Check the [wiki] (https://github.com/se701g2/Doto/wiki/licence) for more details
+Please refer to the [wiki] (https://github.com/se701g2/wiki) for our coding practices 
```

Comment:
Could link directly to https://github.com/se701g2/Doto/wiki/Git-Practices-and-Conventions. Also is this a repeat of above?

Reply:
I just took that out